core/api/workspace.josh.hcl

```diff
@@ -165,17 +171,28 @@ struct "log-description" {
 }
 
 struct "event" {
-	field "id" "string" {}
-	field "log" "string" {}
-	field "timestamp" "string" {}
-	field "message" "string" {}
+  field "id" "string" {}
+  field "log" "string" {}
+  field "timestamp" "string" {}
+  field "message" "string" {}
 }
 
 struct "process-description" {
-	field "id" "string" {}
+  field "id" "string" {}
   field "provider" "string" {}
-	field "name" "string" {}
+  field "name" "string" {}
+  field "running" "bool" {}
+}
+
+struct "component-status" {
```

Comment:
This structure seems to be really _process_ status, not arbitrary component status. Maybe these fields should be added to `process-description`?

Reply:
Ah, yes, I forgot to rename this. I've done that now.